### PR TITLE
gLV 1.27.2 and add pre-prod/preview environments

### PR DESCRIPTION
**Pull request by rdlrt**
- gLV awk print breaks if the sum of CPU stat is too large, replaced with printf.
- Add pre-prod environment to env
- Set SHELLEY_TRANS_EPOCH to 0 for all custom environments
- Add doc updates, rename staging to preprod for confs and change source for downloading configs

### CODE REVIEW

File: scripts/cnode-helper-scripts/env [hunk 1]
```diff
@@ -1017,8 +1017,13 @@ case ${NWMAGIC} in
     NETWORK_NAME="Guild"
     SHELLEY_TRANS_EPOCH=2
     [[ -z ${KOIOS_API} ]] && KOIOS_API="https://guild.koios.rest/api/v0" ;;
+  2)
```
Comment (Scitz0) [inline scripts/cnode-helper-scripts/env:1020]: Remove staging a few lines up?
Nobody runs this right?

**Review by Scitz0 — APPROVED** [review]
Looks good, but see comment for staging. Thoughts?

**Comment by Scitz0** [comment]
And as we are adding pre-prod, should we modify the rest of the files needed for this as well?
Add config, fix prereqs.sh etc?

File: scripts/cnode-helper-scripts/env (hunk 1 above)
Comment (rdlrt) [reply scripts/cnode-helper-scripts/env:1017]: Good point, removed

**Comment by rdlrt** [comment]
> And as we are adding pre-prod, should we modify the rest of the files needed for this as well? Add config, fix prereqs.sh etc?

Agree, added

**Review by Scitz0 — APPROVED** [review]
👍

File: scripts/cnode-helper-scripts/prereqs.sh [hunk 2]
```diff
@@ -470,14 +470,15 @@ if [[ ${NETWORK} = "guild" ]]; then
...
   curl -s -f -m ${CURL_TIMEOUT} -o topology.json.tmp ${URL_RAW}/files/topology-guild.json
   curl -s -f -m ${CURL_TIMEOUT} -o config.json.tmp ${URL_RAW}/files/config-guild.json
-elif [[ ${NETWORK} =~ ^(mainnet|testnet|staging)$ ]]; then
-  curl -sL -f -m ${CURL_TIMEOUT} -o byron-genesis.json.tmp https://hydra.iohk.io/job/Cardano/iohk-nix/cardano-deployment/latest-finished/download/1/${NETWORK}-byron-genesis.json
-  curl -sL -f -m ${CURL_TIMEOUT} -o shelley-genesis.json.tmp https://hydra.iohk.io/job/Cardano/iohk-nix/cardano-deployment/latest-finished/download/1/${NETWORK}-shelley-genesis.json
-  curl -sL -f -m ${CURL_TIMEOUT} -o alonzo-genesis.json.tmp https://hydra.iohk.io/job/Cardano/iohk-nix/cardano-deployment/latest-finished/download/1/${NETWORK}-alonzo-genesis.json
-  curl -sL -f -m ${CURL_TIMEOUT} -o topology.json.tmp https://hydra.iohk.io/job/Cardano/iohk-nix/cardano-deployment/latest-finished/download/1/${NETWORK}-topology.json
-  curl -s -f -m ${CURL_TIMEOUT} -o config.json.tmp ${URL_RAW}/files/config-${NETWORK}.json
+elif [[ ${NETWORK} =~ ^(mainnet|preprod|preview|testnet)$ ]]; then # testnet will be retired , already marked as legacy soon
```
Comment (Scitz0) [inline scripts/cnode-helper-scripts/prereqs.sh:473]: testnet still here

**Review by Scitz0 — COMMENTED** [review]
Preview seem to be missing in doc for a couple of files.
config-preview.json also missing?

File: scripts/cnode-helper-scripts/prereqs.sh (hunk 2 above)
Comment (rdlrt) [reply scripts/cnode-helper-scripts/prereqs.sh:473]: cant remove it completely yet (hence, added comment)...just removed it from cosmetics, atleast until usage decreases and comms are aligned

**Pull request merged**